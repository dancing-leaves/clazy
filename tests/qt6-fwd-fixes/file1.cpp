#include "header1.h"
template <typename Key, typename T> class QCache  ;
class SomeClass;
