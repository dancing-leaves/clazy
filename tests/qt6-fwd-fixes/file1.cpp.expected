qt6-fwd-fixes/file1.cpp:2:1: warning: Using forward declaration of QCache. Including <QtCore/qcontainerfwd.h> instead. [-Wclazy-qt6-fwd-fixes]
